src/libraries/System.Configuration.ConfigurationManager/src/System/Configuration/ConfigurationProperty.cs

```diff
@@ -133,7 +133,14 @@ internal ConfigurationProperty(PropertyInfo info)
 
                 if (collectionAttribute != null)
                 {
-                    if (collectionAttribute.AddItemName.IndexOf(',') == -1) AddElementName = collectionAttribute.AddItemName; // string.Contains(char) is .NetCore2.1+ specific
+#if NETCOREAPP
+                    if (collectionAttribute.AddItemName.Contains(','))
```

Comment:
Same here, probably the reason of the other [test failure](https://github.com/dotnet/runtime/issues/53960)
```suggestion
                    if (!collectionAttribute.AddItemName.Contains(','))
```